rusoto/core/Cargo.toml

```diff
@@ -47,6 +47,8 @@ url = "1.6.0"
 tokio = "0.1.7"
 tokio-timer = "0.2.6"
 xml-rs = "0.7"
+# TODO convert to a feature
+libflate = "0.1"
```

Comment:
I think we should enable it conditionally with a feature flag.

Reply:
Yes, I will create a `compression` feature.